msw/MSW-2D-VERT-03.DATA

```diff
@@ -0,0 +1,1044 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+--
+-- Copyright (c) 2017 SINTEF Digital, Mathematics and Cybernetics.
+-- Copyright (C) 2023 Equinor ASA
+--
+-- This model is a simple three-phase (20, 1, 5) Cartesian regular 2D grid model that users the rock and fluid properties from the
+-- Norne full field model. The model has one standard water injector (INJE01) and one multi-segment horizontal oil well producer
+-- (PROD01). 
+--
+-- Based on MSW-2D-HZ-01. This model tests that the simulator is able to continue if a multi-segment well has no connections to 
+-- the grid (the well will be shut as there are no connections).
+-- 
+-- ( 1) Base model to test a production horizontal multi-segment well (PROD01) in a 2D grid.
+-- ( 2) Maximum well water injection rate set to 1,500 m3/d subject to a maximum BHP of 450 barsa
+-- ( 3) Oil production is controlled by a minimum BHP of 260 barsa.
+-- ( 4) Multi-segment well SUMMARY vectors that are currently supported are now written out.
+--
+-- 456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012
+--       1         2         3         4         5         6         7         8         9         0         1         2         3
+--       0         0         0         0         0         0         0         0         0         1         1         1         1
+-- *********************************************************************************************************************************
+-- =================================================================================================================================
+--
+-- RUNSPEC SECTION
+--
+-- =================================================================================================================================
+RUNSPEC
+--
+--       DEFINE THE TITLE FOR THE RUN
+--
+TITLE
+MSW-2D-HZ: MULTI-SEGMENT WELL 2D GRID HORIZTONAL WELL TEST CASE
```

Comment:
Tiny nit: The case is called `MSW-2D-VERT-03` which suggests a vertical well, but this title says horizontal well.  Did you use [`MSW-2D-HZ`](https://github.com/OPM/opm-tests/blob/b971f56cf305a8b63a79a624ba75c3d36bde0c79/msw/MSW-2D-HZ.DATA) as a template without updating the title/description?

Reply:
Yes.. will fix!